### ISSUE TITLE
Sportmaster_logo_icon_2017.png icon deleted by Wikipedia

### ISSUE THREAD

**Issue by don-vip** [ruosm-presets/literan-moscow#65]
Hello,
The icon for:
```xml
<item name="Sportmaster" ru.name="Спортмастер" type="node,closedway,multipolygon" preset_name_label="true" icon="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/Sportmaster_logo_icon_2017.png/240px-Sportmaster_logo_icon_2017.png">
```
Has been deleted today by Wikipedia for copyright violation:
https://commons.wikimedia.org/wiki/File:Sportmaster_logo_icon_2017.png

> This page has been deleted. The deletion, protection, and move log for the page are provided below for reference.
> 
>     05:47, 10 January 2018 Sealle (talk | contribs) deleted page File:Sportmaster logo icon 2017.png (Copyright violation, see Commons:Licensing) (global usage; delinker log)

Can you please use a new icon?

**Comment by alexander-ii** [comment]
Hi. Thank you for noticing.
Fixed in #66.

**Comment by don-vip** [comment]
thanks for fixing it so fast! :)
